docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -302,13 +302,19 @@ To add Semgrep into your CircleCI pipeline:
 1. Create a [context](https://circleci.com/docs/contexts/):
     1. In CircleCI web app, click **Organization Settings** > **Contexts**. 
     2. Click **Create Context**.
-    3. Enter `dev` as the name for the context.
+    3. Enter `semgrep` as the name for the context.
     4. Click **Add Environment Variable** and enter your `SEMGREP_APP_TOKEN`.
 2. Create or edit your `config.yml` configuration file in the repository you want to scan.
 3. Copy the relevant code snippet provided in [Sample CircleCI configuration snippet](#sample-circleci-configuration-snippet).
 4. Commit the updated `config.yml` configuration file into the `/.circleci` folder in the target repository.
 5. The Semgrep job starts automatically upon detecting the `config.yml` update.
-6. Optional: Create a separate CI job for diff-aware scanning, which scans only changed files in PRs or MRs, by repeating steps 1-3 and uncommenting the `SEMGREP_BASELINE_REF` definition provided in the code snippet.
+6. This configuration is ready for full scanning or diff-aware scanning, which scans only changed files in PRs or MRs. 
+
+CircleCI by default will build all the commits for the project. If you want to build only branches that have associated a pull request open, you can enable the option "Only build pull requests" in **Project Settings** > **Advanced**.
+
+:::note
+For the default branch and tags, CircleCI will always build all commits.
+:::
```

Comment:
@sebasrevuelta 

6 says "full scanning or diff aware scanning" - do you mean "full scanning **AND** diff-aware scanning"? i.e. the CI job can do both without the need for additional jobs?

Reply:
You are right @s-santillan , I will change it.